src/snowflake/connector/connection.py

```diff
@@ -1182,6 +1182,11 @@ def __config(self, **kwargs):
             ]:
                 self._authenticator = auth_tmp
 
+        # read OAuth token from
```

Comment:
you might want to add doc/default value for it:

doc: https://github.com/snowflakedb/snowflake-connector-python/blob/main/src/snowflake/connector/connection.py#L310

and default value: https://github.com/snowflakedb/snowflake-connector-python/blob/main/src/snowflake/connector/connection.py#L157

Reply:
We can just reuse `token`

Reply:
Default value may not be required because we will reuse `token`